neuralop/layers/tests/test_padding.py

```diff
@@ -3,16 +3,44 @@
 import pytest
 
 @pytest.mark.parametrize('mode', ['one-sided', 'symmetric'])
-def test_DomainPadding(mode):
-    out_size = {'one-sided': 12, 'symmetric': 14}
+@pytest.mark.parametrize('padding_dim', [0.2, [0.1, 0.2]])
+def test_DomainPadding_2d(mode, padding_dim):
+    if isinstance(padding_dim, float):
+        out_size =  {'one-sided': [12, 12], 'symmetric': [14, 14]}
+    else:
+        out_size =  {'one-sided': [11, 12], 'symmetric': [12, 14]} 
+
     data = torch.randn((2, 3, 10, 10))
-    padder = DomainPadding(0.2, mode)
+    padder = DomainPadding(padding_dim, mode)
     padded = padder.pad(data)
 
     target_shape = list(padded.shape)
-    target_shape[-1] = target_shape[-2] = out_size[mode]
+    for ctr in range(1,3):
+        target_shape[-ctr] = out_size[mode][-ctr]
     assert list(padded.shape) == target_shape
 
     unpadded = padder.unpad(padded)
     assert unpadded.shape == data.shape
 
+
+@pytest.mark.parametrize('mode', ['one-sided', 'symmetric'])
+@pytest.mark.parametrize('padding_dim', [0.2, [0.1, 0, 0.2]])
+def test_DomainPadding_3d(mode, padding_dim):
+    if isinstance(padding_dim, float):
+        out_size =  {'one-sided': [12, 12, 12], 'symmetric': [14, 14, 14]}
+    else:
+        out_size =  {'one-sided': [11, 10, 12], 'symmetric': [12, 10, 14]} 
+
+    data = torch.randn((2, 3, 10, 10, 10))
+    padder = DomainPadding(padding_dim, mode)
+    padded = padder.pad(data)
+
+    target_shape = list(padded.shape)
+    for ctr in range(1,4):
```

Comment:
Same comment as above - ideally tests should be self explanatory and easy to get by just reading the code!